package.json

```diff
@@ -5,7 +5,7 @@
   "main": "dist/rollup.js",
```

Comment:
This change also means that we can–and should–move all `dependencies` back into `devDependencies` so that rollup is now back to its small and lean footprint!